docs/api.rst

```diff
@@ -2170,8 +2170,23 @@ this goal, it must make use of a couple of data classes that aid in this goal.
 
         The type of channel or channel permission overwrite.
 
-        If the type is an :class:`int`, then it is a type of channel which can be either
-        ``0`` to indicate a text channel or ``1`` to indicate a voice channel.
+        If the type is an :class:`int`, then it is one of the following:
+
+        +-----+---------------------------------+
+        | `0` | A guild text channel.           |
```

Comment:
I meant like \``0``

Reply:
Oh right sorry I am not familiar with rst